NOTICE.adoc

```diff
@@ -0,0 +1,21 @@
+= Notices for the MuleSoft Documentation UI Project
+
+This UI for Antora is produced and maintained by the MuleSoft Documentation team.
+
+== Copyright
+
+This UI is derived from the https://gitlab.com/antora/antora-ui-default/[Antora Default UI].
+Modifications have been applied to align the theme and behavior of the UI with the MuleSoft brand and documentation requirements.
+All branding elements in this UI, including icons, images, and fonts, are copyright Salesforce.
```

Comment:
I'd change this to:

MuleSoft brand elements, including icons, images, and fonts, and the visual design of this UI are copyright MuleSoft LLC, a Salesforce company. All rights to the brand elements and visual design are reserved and not licensed under the terms mentioned below.